articles/android-studio-shimmer-loading-effect/index.md

```diff
@@ -283,9 +289,9 @@ Here is the shimmer layout design.
 
```

Comment:
android:orientation="vertical" does not have an effect on the parent RelativeLayout. Its mainly for LinearLayout. Could you please remove it.